src/Components/VideoSearchList.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { ListGroup, Image, Row, Col } from "react-bootstrap/";
+
+const VideoSearchList = (props) => {
+  const listClickHandler = (event) => {

```

Comment:
`handleListClick` would be a better name as mentioned above, functions should be verbs.